app/views/layouts/home.html.erb

```diff
@@ -1,5 +1,8 @@
 <% content_for :script do %>
   <%= render "home/shared/scripts" %>
+  <% if ENV["HELPER_AI_MAILBOX_SLUG"].present? %>
+    <script src="https://helper.ai/widget/sdk.js" data-mailbox="<%= ENV["HELPER_AI_MAILBOX_SLUG"] %>" async></script>
```

Comment:
`app/views/layouts/shared/_helper_widget.html.erb` is what we use to load the widget at the moment (it's not using the latest integration code but should still work), so I think the solution is to edit `show_helper_widget?` not add a second integration